modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/creating-first-moa-cluster.adoc
+
+
+[id="moa-installing-prerequisites"]
+= Installing prerequisites
+
+Complete the following prerequisites before creating your {product-title} cluster.
+
+.Procedure
+
+. Select the AWS account you want to use.
++
+Unless you are just testing out MOA, we recommend using a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
```

Comment:
Unless this is a style exception, we avoid using _we_ when making recommendations.